CONTRIBUTING.md

```diff
@@ -1,21 +1,23 @@
 We happily welcome contributions to Delta Lake. We use [GitHub Issues](/../../issues/) to track community reported issues and [GitHub Pull Requests ](/../../pulls/) for accepting changes.
 
 # Governance
-Delta Lake is an independent open-source project and not controlled by any single company. To emphasize this we joined the Delta Lake Project in 2019, which is a sub-project of the Linux Foundation Projects. Within the project, we make decisions based on [these rules](https://delta.io/pdfs/delta-charter.pdf).
+Delta Lake is an independent open-source project and not controlled by any single company. To emphasize this we joined the [Delta Lake Project](https://community.linuxfoundation.org/delta-lake/) in 2019, which is a sub-project of the Linux Foundation Projects. Within the project, we make decisions based on [these rules](https://delta.io/pdfs/delta-charter.pdf).
 
-Delta Lake is supported by a wide set of developers from over 50 organizations across multiple repositories.  Since 2019, more than 190 developers have contributed to Delta Lake!  The Delta Lake community is growing by leaps and bounds with more than 5500 members in the [Delta Users slack](https://dbricks.co/delta-users-slack)).
+Delta Lake is supported by a wide set of developers from over 50 organizations across multiple repositories.  Since 2019, more than 190 developers have contributed to Delta Lake!  The Delta Lake community is growing by leaps and bounds with more than 6000 members in the [Delta Users slack](https://dbricks.co/delta-users-slack)).
 
 For more information, please refer to the [founding technical charter](https://delta.io/pdfs/delta-charter.pdf).
 
 # Communication
-- Before starting work on a major feature, please reach out to us via [GitHub](https://github.com/delta-io/delta/issues), [Slack](https://dbricks.co/delta-users-slack), [email](https://groups.google.com/g/delta-users), etc. We will make sure no one else is already working on it and ask you to open a GitHub issue.
+- Before starting work on a major feature, please reach out to us via [GitHub](https://github.com/delta-io/delta/issues), [Slack](https://delta-users.slack.com/join/shared_invite/zt-165gcm2g7-0Sc57w7dX0FbfilR9EPwVQ#/shared-invite/email), [email](https://groups.google.com/g/delta-users), etc. We will make sure no one else is already working on it and ask you to open a GitHub issue.
```

Comment:
do we have a long term solution for slack link? It will expire and in this case we want to update it quickly.

Reply:
- In the last workshop, we gave [this link]( https://delta-users.slack.com/) and it didn't work for users who never joined slack.
- So we gave [this link](https://join.slack.com/t/delta-users/shared_invite/zt-165gcm2g7-0Sc57w7dX0FbfilR9EPwVQ) to the new users who wanted to join slack.

Happy to use what you think would make sense @zsxwing.

Reply:
Just checked `https://dbricks.co/delta-users-slack`. Currently it points to `https://join.slack.com/t/delta-users/shared_invite/zt-165gcm2g7-0Sc57w7dX0FbfilR9EPwVQ`. I assumed @dennyglee updated the pointed link regularly to keep `https://dbricks.co/delta-users-slack` always valid without updating this page.

If we switch back to `https://delta-users.slack.com/join/shared_invite/zt-165gcm2g7-0Sc57w7dX0FbfilR9EPwVQ#/shared-invite/email`, we will have to update this page every time when this link is about to expire.

Reply:
Note, please update the link to https://go.delta.io/delta-users - thanks!

Reply:
Yes. this is the link the website uses as well. thank you.